frontend/src/components/tabs/RuntimeNodeDetailsV2.tsx

```diff
@@ -103,17 +116,34 @@ export function RuntimeNodeDetailsV2({
 }
 
 interface TaskNodeDetailProps {
+  runId?: string;
   element?: FlowElement<FlowElementDataBase> | null;
   execution?: Execution;
   namespace: string | undefined;
 }
 
-function TaskNodeDetail({ element, execution, namespace }: TaskNodeDetailProps) {
+function TaskNodeDetail({ runId, element, execution, namespace }: TaskNodeDetailProps) {
+  const { data: logsInfo } = useQuery<Map<string, string>, Error>(
+    ['log_details'],
+    async () => {
+      if (!execution) {
+        throw new Error('No execution is found.');
+      }
+      return await getLogsInfo(execution, runId);
+    },
+    { enabled: !!execution },
+  );
+
+  const logsDetails = logsInfo?.get('logsDetails');
+  const logsBannerMessage = logsInfo?.get('logsBannerMessage');
+  const logsBannerAdditionalInfo = logsInfo?.get('logsBannerAdditionalInfo');
```

Comment:
`'logsDetails'`, `'logsBannerMessage'` and `'logsBannerAdditionalInfo'` are constant keys which should be extracted to global const, so you can reuse these keys in both the caller and within the `getLogsInfo` function.

And if there is a change in the future, we make sure callers and function are consistent as well.